src/App.js

```diff
@@ -1,7 +1,77 @@
+import React from 'react';
 import "./App.css";
+import ProductData from './data/productData';
+import Products from './Components/Products';
+import AddToCart from './Components/AddToCart';
+import Checkout from './Components/Checkout';
 
-const App = () => {
-  return <h1>Hello, world!</h1>;
+class App extends React.Component {
+  constructor() {
+    super()
+    this.state = {
+      cart: [],
+      subtotal: '',
+      tax: '',
+      total: ''
```

Comment:
Since these values can be directly calculated based on the items inside the cart, these values don't belong in state.
Instead, try calculating subtotal, tax, and total at the beginning of your render method.